contracts/governance/Vesting/VestingRegistryStorage.sol

```diff
@@ -61,4 +61,9 @@ contract VestingRegistryStorage is Initializable, AdminRole {
     ///@notice A record of all vesting addresses
     ///@dev isVesting[address] returns if the address is a vesting address
     mapping(address => bool) public isVesting;
+
+    ///@notice A record of all vesting addresses with the detail
+    ///@dev vestingDetail[vestingAddress] returns Vesting struct data
+    ///@dev can be used to easily check the vesting type / creation type based on the vesting address itself
+    mapping(address => Vesting) public vestingDetail;
```

Comment:
```suggestion
mapping(address => Vesting) public vestingDetails;
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/79f72c867c17cd1f12ed1e726ee79e2c308a8e9e)